src/js/core.js

```diff
@@ -268,11 +268,33 @@ function MediumEditor(elements, options) {
         return shouldAdd;
     }
 
+    function createContentEditable(index) {
+        var div = this.options.ownerDocument.createElement('div');
+        var id = (+new Date());
+        var textarea;
+        div.innerHTML = this.elements[index].value;
+        div.setAttribute('data-textarea', id);
+        this.elements[index].classList.add('medium-editor-hidden');
+        this.elements[index].setAttribute('data-textarea', id);
+        this.elements[index].parentNode.insertBefore(
+            div,
+            this.elements[index]
+        );
+        textarea = this.options.ownerDocument.querySelectorAll('textarea[data-textarea="' + id + '"]')[0];
```

Comment:
probably don't have to go re-lookup this? just get a ref before you replace the `elements[index]` no?